Include/internal/pycore_long.h

```diff
@@ -130,8 +130,8 @@ PyAPI_FUNC(char*) _PyLong_FormatBytesWriter(
  * we define them to the numbers in both places and then assert that
  * they're the same.
  */
-static_assert(SIGN_MASK == _PyLong_SIGN_MASK);
-static_assert(NON_SIZE_BITS == _PyLong_NON_SIZE_BITS);
+static_assert(SIGN_MASK == _PyLong_SIGN_MASK, "SIGN_MASK does not match _PyLong_SIGN_MASK");
+static_assert(NON_SIZE_BITS == _PyLong_NON_SIZE_BITS, "NON_SIZE_BITS does not match _PyLong_NON_SIZE_BITS");
```

Comment:
I'm curious why this is best practice. The message simply repeats the expression; if it was a comment it'd be redundant.
Are there compilers that don't show the source line when a static assertion fails?

Reply:
What message would you prefer?

Reply:
No problem. I always forget about C++, so it balances out 🙂